core/src/nusi/wrapped-tag.tsx

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2021 Terminus, Inc.
+//
+// This program is free software: you can use, redistribute, and/or modify
+// it under the terms of the GNU Affero General Public License, version 3
+// or later ("AGPL"), as published by the Free Software Foundation.
+//
+// This program is distributed in the hope that it will be useful, but WITHOUT
+// ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+// FITNESS FOR A PARTICULAR PURPOSE.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+import React from 'react';
+import { Tag } from 'antd';
+import { TagProps } from 'antd/lib/tag';
+
+// Regular expression for hexadecimal colors
```

Comment:
to be more detailed.  such as `to match #xxxxxxx`  
also, rename this variable to more specific word like `colorRegex`